frontend/src/screens/apps/ShowApp.tsx

```diff
@@ -416,6 +441,18 @@ function AppInternal({ app, refetchApp, capabilities }: AppInternalProps) {
               </CardTitle>
             </CardHeader>
             <CardContent>
+              <div className="flex justify-end text-sm text-muted-foreground mb-4">
```

Comment:
Note: you should try to be consistent with our other buttons in the app, and use existing components where possible